native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -0,0 +1,86 @@
+import { Platform } from 'react-native';
+
+const IOS_TOKENS = require('bpk-tokens/tokens/ios/base.react.native.common.js');
+const ANDROID_TOKENS = require('bpk-tokens/tokens/android/base.react.native.common.js');
+
+const tokens = Platform.select({
+  ios: () => IOS_TOKENS,
+  android: () => ANDROID_TOKENS,
+})();
+
+const styles = {
+  base: {
+    underlayColor: 'rgba(0, 0, 0, 0.15)',
+    text: {
+      backgroundColor: 'transparent',
+      color: tokens.colorWhite,
+      fontSize: tokens.textSmFontSize,
```

Comment:
Create a layer of indirection for tokens don't directly reference the bonds. This should be `tokens.buttonTextFontSize`and so on for all other tokens in this component

Reply:
Will do - it's on my todo list